### PR TITLE
Keybase and Bitwarden services added

### PR DESCRIPTION
**Pull request by Ampsicora**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,434 +131,436 @@ Table of Contents
 
 ## Source Code Repos
 
-  * [Azure DevOps](https://azure.microsoft.com/services/devops/) — Unlimited private repos (Git and TFS) for up to 5 users/team
```
Comment (ripienaar) [inline README.md]: why remove all these?

Comment (iondrey) [reply README.md]: I think this is a desire to make a single format

Comment (ripienaar) [reply README.md]: Judging by the size of the commit it was already pretty uniform :P and adding the / means fewer redirects when visiting the sites.

Comment (Ampsicora) [reply README.md]: > why remove all these?

Removing the "/"  was just a proposal to better uniform links, feel free to ignore it if you prefer

Comment (ripienaar) [reply README.md]: You'd need to remove it from the PR, I cant selectively ignore that

**Pull request closed without merging**